backend/src/main/java/com/huellapositiva/application/dto/ChangePasswordDto.java

```diff
@@ -0,0 +1,25 @@
+package com.huellapositiva.application.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ChangePasswordDto {
+
+    @NotEmpty
+    @Size(min = 6, max = 15)
+    @Pattern(regexp = "^[a-zA-Z0-9]*$", message = "Nueva contrasenna falla")
+    private final String newPassword;
+
+    @NotEmpty
+    @Size(min = 6, max = 15)
```

Comment:
Why wouldn't we allow more than 15 characters?
Why don't we allow special characters? Passwords would be more secure and much much less vulnerable to brute force attacks

Reply:
Ok, i change it right now. 

The special characters to add "-.,:;+-*/!@#$&" was right? there wasnt problems with accents? (áàäâã)

Reply:
Done